mungegithub/mungers/approval-handler.go

```diff
@@ -281,13 +307,18 @@ func removeSubdirs(dirList []string) sets.String {
 }
 
 // getMessage returns the comment body that we want the approval-handler to display on PRs
+// Params
+// 	- obj: mungeObj representing the PR
+// 	- ownersMap: map from list of files to people that have approved
+// 	- alreadyApproved: set containing ALL the people that have approved the PR
+// 	- isFullyApproved: bool indicating if PR fully approved or NOT
 // The comment shows:
 // 	- a list of approvers files (and links) needed to get the PR approved
 // 	- a list of approvers files with strikethroughs that already have an approver's approval
 // 	- a suggested list of people from each OWNERS files that can fully approve the PR
 // 	- how an approver can indicate their approval
 // 	- how an approver can cancel their approval
-func (h *ApprovalHandler) getMessage(obj *github.MungeObject, ownersMap map[string]sets.String, alreadyApproved sets.String, isFullyApproved bool) string {
+func (h *ApprovalHandler) getMessage(obj *github.MungeObject, ownersMap map[string]sets.String, alreadyApproved, suggestedApprovers sets.String, isFullyApproved bool) string {
```

Comment:
This as grown out of sanity. Make a mental note that it would be nice to rewrite that somehow next time you touch this piece of code.